set-web/src/main/java/eu/europeana/set/web/model/search/CollectionPage.java

```diff
@@ -26,14 +28,15 @@ public int getStartIndex() {
     public void setStartIndex(int startIndex) {
         this.startIndex = startIndex;
     }
-    
-    public void setSet(UserSet set) {
-        this.set = set;
+
+    @JsonProperty(WebUserSetFields.ITEMS)
+    @JsonRawValue
```

Comment:
this breaks the standard profile, we probably need to separate the class hierarchy and Create ItemIdsCollectionPage and ItemDescriptionsCollectionPage, where we can apply the @JsonRawValue for the serialization in the later one.

Reply:
created the separate classes